### DISCUSSION TITLE
Welcome to developer-portfolio Discussions!

### DISCUSSION THREAD

**Discussion by said7388** [said7388/developer-portfolio#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yogithesymbian** [comment]
Hi @said7388 ,

how about dark / light theme or connecting section for blog to for example `blogger` or `wordpress` so its autmatically pagination for post and when click its directly open new tab browser to the post 🤙 or do you have other option or suggestion 🎸

**Comment by adriangrahldev** [comment]
Hello, I get an error with nodejs 21. I get the error document is not defined. I think it is due to the lottie animations module, in the createTag function

**Reply by Maahdi313** [reply]
Hey did you fix the issue, i have the same problem, if you solved it please tell me what you did. 
<img width="683" alt="error" src="https://github.com/user-attachments/assets/7f4eba60-405d-486a-8305-577348055501" />
